content/en/docs/concepts/services-networking/service.md

```diff
@@ -139,7 +139,7 @@ spec:
   - name: name-of-service-port
     protocol: TCP
     port: 80
-    targetPort: http-web-service
+    targetPort: http-web-svc
```

Comment:
verified: If it is a string, it must be not empty and no more than 15 characters. It must contain only alphanumeric characters and hyphens.